### PR TITLE
Jump to 2.0.0

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Make sure that the hub works with the latest version of z2j 
- Update base image
- Require newer kubespawner
- Fix async calls 

---

**Related issue :**

### CODE REVIEW

**Comment by enolfc** [comment]
Testing in progress at training notebooks

**Comment by enolfc** [comment]
This is deployed at training.notebooks.egi.eu and apparently working fine.
There are possible improvements here and there for the future (refactor code related to sidecar storage to avoid repetition, further debug refresh of tokens)

**Comment by enolfc** [comment]
There is one issue with the B2DROP code as it has async call in a non async function

**Comment by enolfc** [comment]
Fails for the monitoring user as the secret is not available there

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 jupyterhub>=1.2
 oauthenticator>=14.0.0
 jupyterhub-kubespawner>=4.3.0
+kubernetes
```
Comment (enolfc) [inline requirements.txt:4]: should this be kubernetes-asyncio instead?

Comment (valtri) [reply requirements.txt:4]: Oh, I see. I guess it was just an issue with forgotten jupyterhub version "1.1.3" in k8s and this dependency is not needed at all.

**Comment by enolfc** [comment]
@andrea-manzi any comments on this one?

**Comment by andrea-manzi** [comment]
i don't have comments but if @valtri did test the new version and he found no issuess better if he approves the changes

**Review by valtri — APPROVED** [review]
Tested and it looks good. I couldn't test onedata hooks in testing environment, but I wouldn't expect problems. Also onedata is properly skipped when not used (for example locally implemented B2DropSpawner can be inherited from OnedataSpawner and without onedata-specific configuration).

**Pull request merged**